pkg/apis/pingcap/v1alpha1/tidbinitializer_types.go

```diff
@@ -53,21 +54,27 @@ type TidbInitializer struct {
 // +k8s:openapi-gen=true
 // TidbInitializer spec encode the desired state of tidb initializer Job
 type TidbInitializerSpec struct {
+	Image string `json:"image"`
```

Comment:
The `Image` and `ImagePullPolicy` are to cover below configs in `values.yaml`, right?
```
# mysqlClient is used to set password for TiDB
# it must has Python MySQL client installed
mysqlClient:
  image: tnir/mysqlclient
  imagePullPolicy: IfNotPresent
```
We also have to cover the initializer.resources config:
```
  initializer:
    resources: {}
    # limits:
    #  cpu: 100m
    #  memory: 100Mi
    # requests:
    #  cpu: 100m
    #  memory: 100Mi
```

Reply:
Good catch!